string pickUp
string deliver
---
bool status
